src/player_activity.cpp

```diff
@@ -158,11 +158,18 @@ void player_activity::do_turn( player &p )
         p.drop_invalid_inventory();
         return;
     }
-
+    const bool travel_activity = id() == activity_id( "ACT_TRAVELLING" );
```

Comment:
```suggestion
    const bool travel_activity = id() == ACT_TRAVELLING;
```

Reply:
You would possibly need to add `#include "cata_string_consts.h"` on top as well.